hadoop-hdds/rocksdb-checkpoint-differ/src/main/java/org/apache/ozone/rocksdb/util/RdbUtil.java

```diff
@@ -58,4 +65,17 @@ public static Set<String> getSSTFilesForComparison(final String dbLocation,
     }
   }
 
+  public static <T> Stream<T> getStreamFromIterator(
```

Comment:
I don't think it has to be a Util as of now and can be internal to `ManagedSstFileReader`.

Same goes for `getSSTFilesForComparison` but you can leave that as it is for now.